### PR TITLE
Revert "Update homepage grid, should have been in #4"

### PR DESCRIPTION
**Pull request by Gerald12344**
Reverts Edulink-2-0/Edulonk#5 someone clicked merge on my PC

### CODE REVIEW

**Comment by Gerald12344** [comment]
Ok reversed its chill now :)

File: components/homepage/homepageGrid.tsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ export default function HomepageGrid({
                 <Button variant="outline">
                     <Text
                         gradient={{
-                            from,
```
Comment (Gerald12344) [inline components/homepage/homepageGrid.tsx:27]: In theory this should work cause JS is weriddddd, but haven't test, can someone test then review pls.

Thanks,
Harvey

Comment (20c33406) [reply components/homepage/homepageGrid.tsx:27]: It looks fine (It looks exactly the same)

Seth

**Comment by Gerald12344** [comment]
Jack u have 30 seconds to approve my PR or I will freak out

**Pull request merged**